### PR TITLE
[batch]  clean up batch/ now that batch_client has been broken out

### PR DESCRIPTION
**Pull request by cseed**
Also clean up some pylint warnings.

### CODE REVIEW

File: batch/setup.py [hunk 1]
```diff
@@ -6,19 +6,6 @@
     url = 'https://github.com/hail-is/hail.git',
     author = 'Hail Team',
     author_email = 'hail@broadinstitute.org',
-    description = 'Kubernetes job manager client',
-    packages = find_packages(),
-    include_package_data=True,
-    install_requires=[
-        'cerberus',
-        'kubernetes',
-        'flask',
```
Comment (jigold) [inline batch/setup.py]: Where did these requirements go?

Comment (cseed) [reply batch/setup.py]: So it seems like dependencies are duplicated: here and the requirements.txt for base.  I decided to start specifying just the dependencies in base.  Many packages are used in different projects, so it seemed like there was less duplication if base requirements.txt was the reference.

It's not just the packages, we're pinning packages now, so versions will be duplicated (and therefore probably out of sync), see: https://github.com/hail-is/hail/commit/fab8d834b5b252013d8c077915ea4e2a9cd33a71

File: batch/Makefile [hunk 2]
```diff
@@ -40,7 +40,7 @@ check: flake8-stmp pylint-stmp
 build:
 	make -C ../docker build
 	-docker pull $(BATCH_LATEST)
-	python3 ../ci/jinja2_render.py '{"base_image":{"image":"base"}}' Dockerfile Dockerfile.out
+	python3 ../builder/jinja2_render.py '{"base_image":{"image":"base"}}' Dockerfile Dockerfile.out
```
Comment (jigold) [inline batch/Makefile]: This got reverted -- sorry about that...

**Comment by cseed** [comment]
No worries, I was expecting that.  Fixed.

**Pull request merged**